components/arm/universalrobots/ur.go

```diff
@@ -240,6 +246,9 @@ func (ua *URArm) MoveToPosition(
 ) error {
```

Comment:
Per Eliot at our last meeting, we don't want this to be a robot config value, but something passed in via the `extra` parameter.

Reply:
After talking with Eliot, we're just going to do both, so that you have the option of doing either